src/terrasoBackend/api.js

```diff
@@ -9,18 +10,26 @@ const handleGraphQLError = data => {
 }
 
 export const request = async (query, variables) => {
-  const response = await fetch(TERRASO_API_URL, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json'
-    },
-    body: JSON.stringify({ query, variables })
-  })
+  const response = await fetch(
+    new URL('/graphql/', TERRASO_API_URL).href,
```

Comment:
Q: Should `/graphql/` be in a constant somewhere?

Reply:
It will only be used here, it can be a constant in this same file, but I don't see us changing this ever so maybe it is better to have it directly here, but let me know what do you think

Reply:
Would still be good to put in a config section somewhere.